create/remote-pipelines-tasks-resolvers.adoc

```diff
@@ -1,7 +1,7 @@
 :_mod-docs-content-type: ASSEMBLY
 include::_attributes/common-attributes.adoc[]
 [id="remote-pipelines-tasks-resolvers"]
-= Specifying remote pipelines and tasks using resolvers
+= Specifying remote pipelines, tasks, and step actions using resolvers
```

Comment:
```suggestion
= Specifying remote pipelines, tasks, and StepActions using resolvers
```

Reply:
The style guide does not permit us to use StepAction as a standalone noun. It can be a StepAction definition, StepAction resource, and so on, but when it's a standalone thing, it's "step action". (And plurals of resource names, such as PipelineRuns or StepActions, can also be confusing)